appium/webdriver/appium_connection.py

```diff
@@ -26,55 +25,33 @@
 
 PREFIX_HEADER = 'appium/'
 
+_HEADER_IDEMOTENCY_KEY = 'X-Idempotency-Key'
```

Comment:
why the constant should be private?

Reply:
yes, I don't see any harm when people could import this header name

Reply:
Sure, I don't have strong opinion to make this private/public. Let me update